### ISSUE TITLE
Filter suggestions don't work if your model name is different from your project name

### ISSUE THREAD

**Issue by simond** [looker-open-source/components#2925]
When using the `<DashboardFilter>` component, suggestions never work if your model name is different from your project name. Reproduce with the following setup: 

- Create a project called `my-project-name` 
- Create a model called `my-model-name.model` 
- Create a user-defined dashboard with a single filter on it and get the dashboard id - in my case the dashboard id is 153
- Create an extension app and try to add a `<DashboardFilter>` that points to the filter on the above dashboard

## Project Setup
![image](https://user-images.githubusercontent.com/421551/145679564-1d8e6b09-e55f-4f03-b6d6-b7305a444f43.png)

## Dashboard Setup
![image](https://user-images.githubusercontent.com/421551/145679774-9a4c00c2-b7b3-4efc-a405-07c88b33d749.png)

## Extension Framework Code
Here is the extension framework code I am trying to use to add the filter to a blank page. I fetch the filter from dashboard created above (id = 153) and then try to render the filter to the page. The filter gets rendered correctly but the suggestions do not show:
```javascript
import React, {useEffect, useState, useContext} from 'react'
import {ComponentsProvider} from '@looker/components'
import {ExtensionContext2} from '@looker/extension-sdk-react'
import {DashboardFilter} from '@looker/filter-components'

export const FiltersDemo = () => {
  const [filter, setFilter] = useState(undefined)
  const {coreSDK, extensionSDK} = useContext(ExtensionContext2)

  // Get the first filter from sample dashboard 153 and od it into the 'filter' state
  useEffect(() => {
    const dashboard = '153';
    const loadFilter = async (id) => {
      const result = await coreSDK.ok(coreSDK.dashboard(id));
      const filter = result.dashboard_filters[0]
      setFilter(filter);
    };

    loadFilter(dashboard);
  }, [coreSDK])

  return (
    <ComponentsProvider>
      <div style={{width: '200px'}}>
        {filter ?
          <DashboardFilter
            key={filter.id}
            sdk={coreSDK}
            filter={filter}
          /> : 'Loading Filter'
        }
      </div>
    </ComponentsProvider>
  )
}
```
I get the following error on the filter and the suggestions do not show.
![image](https://user-images.githubusercontent.com/421551/145679878-5136c0fa-32f6-41d2-8d0e-9bbb42f38aba.png)

I think the problem is in the `useSuggestable` hook. [Here](https://github.com/looker-open-source/components/blob/main/packages/filter-components/src/DashboardFilter/use_suggestable.ts#L94) you can see that the `field.project_name` is being passed through to `model_fieldname_suggestions` instead of the `filter.model`. 

```javascript
  model_fieldname_suggestions(sdk, {
    field_name: field?.suggest_dimension || '',
    model_name: field?.project_name || '', // <-------- issue here?
    term: searchTerm,
    view_name: field?.suggest_explore || field?.view || '',
  })
```

**Comment by mdodgelooker** [comment]
Thank you @simond ! We'll get a fix up for this ASAP.

I'm not sure if this would be of interest for your use case, but the `Filter` component is also available for direct use. It's the underlying component in `DashboardFilter`, minus the hooks `useSuggestable`, `useExpressionState`, and `useValidationMessage` and the `Field` component from `@looker/components`, each of which can be included as needed. "Advanced usage" if you will. 🙂

**Comment by simond** [comment]
Thanks @mdodgelooker . Actually, I like the DashboardFilter abstraction so I just copied your code and made the fix in the useSuggestable hook myself. When this bug gets resolved I'll switch back to using the one from this repo!!

**Comment by mdodgelooker** [comment]
@simond that works! 👍

**Comment by mdodgelooker** [comment]
Hi @simond the fix is published in `@looker/filter-components@0.10.13`.

**Comment by simond** [comment]
> Hi @simond the fix is published in `@looker/filter-components@0.10.13`.

Awesome, thanks @mdodgelooker !!
